### PR TITLE
Fix rspec eventually bug

### PR DESCRIPTION
**Pull request by adsteel**
Created from https://github.com/mojotech/dill/pull/103

### CODE REVIEW

**Comment by david** [comment]
:shipit: (feel free to merge @adsteel)

**Comment by adsteel** [comment]
@david Hmm... It doesn't like that this error class is defined. I can't `require 'rspec'` either. Any suggestions? I could store the classes as strings and convert them to classes in the rescue block, but that seems a bit dirty.

**Comment by david** [comment]
@adsteel perhaps you could add that class only if `RSpec` is defined? I'd rather avoid using strings.

File: lib/dill/checkpoint.rb [hunk 1]
```diff
@@ -13,6 +13,10 @@ class << self
 
     self.rescuable_errors = [StandardError]
 
+    if defined?(RSpec::Expectations)
```
Comment (adsteel) [inline lib/dill/checkpoint.rb]: @david What do you think of this?

Comment (david) [inline lib/dill/checkpoint.rb]: That works.

**Pull request merged**